.github/workflows/post-deploy.yml

```diff
@@ -133,7 +133,7 @@ jobs:
         run: python -m pip install --upgrade pip && python -m pip install -r requirements-dev.txt bandit==1.7.4
       - name: Bandit
         working-directory: ${{ inputs.app_name}}
-        run: bandit -r ./app
+        run: bandit -r . -lll
```

Comment:
Am I reading/interpreting correctly that this step runs after the code has already been merged to main, or? That feels like the wrong time to be running this step because it'll block the pipeline.

(Aware this would clearly be a pre-existing issue 👀)

Reply:
I thought the same.  

We should take a look at those workflows.